### PR TITLE
Add CNY/LNY 2018 newspost and assets

### PR DESCRIPTION
**Pull request by Ephemeralis**
Lunar New Year newspost pending possible design changes, need someone to vet the Korean line if possible!

### CODE REVIEW

File: news/2018-02-16-happy-lunar-new-year.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+2018년 정유년 새해 복 많이 받으시고, 행복한 일만 가득한 새해 되세요!
+
+Selamat Tahun Baru Imlek 2018,  Semoga anda Makmur di Tahun Baru ini!
+
+Selamat Tahun Baru Cina 2018, Salam Tahun Baru Yang Penuh Dengan Kemakmuran!
+
+[![](/wiki/shared/news/2018-02-16-happy-lunar-new-year/cny2018-poster.jpg)](/wiki/shared/news/2018-02-16-happy-lunar-new-year/cny2018-poster.jpg)
+
+We're proud to support and be the home of one of the most vibrant and thriving Asian communities on the internet. Remember to give your friends your well wishes, and go collect those red packets!
```
Comment (TPGPL) [inline news/2018-02-16-happy-lunar-new-year.md]: Well wishes?

File: news/2018-02-16-happy-chinese-new-year.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+layout: post
+title: "Happy Chinese New Year!"
+date: 2018-02-16 05:00:00 +0000
+---
+
+汪汪! The world wags its tail in anticipation of the Year of the Dog. The osu!team would like to wish everyone a year of boundless fortunes, overflowing happiness, and great success in whatever challenges you may face.
+
+![](/wiki/shared/news/2018-02-16-happy-chinese-new-year/header.jpg)
+
+辞旧灵鸡歌日丽 ，迎新瑞犬报年丰，犬效丰年家家富 ，鸡鸣盛世处处春
```
Comment (kj415j45) [inline news/2018-02-16-happy-chinese-new-year.md]: >辞旧灵鸡歌日丽，迎新瑞犬报年丰。犬效丰年家家富 ，鸡鸣盛世处处春。

maybe better? @osu-translate-zh

File: news/2018-02-16-happy-chinese-new-year.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+辭舊靈雞歌日麗，迎新瑞犬報年豐，犬效豐年家家富，雞鳴盛世處處春
```
Comment (kj415j45) [inline news/2018-02-16-happy-chinese-new-year.md]: >辭舊靈雞歌日麗，迎新瑞犬報年豐。犬效豐年家家富，雞鳴盛世處處春。

ditto

File: news/2018-02-16-happy-chinese-new-year.md (hunk 2 above)
Comment (Deardrops) [reply news/2018-02-16-happy-chinese-new-year.md]: ok

**Pull request merged**